web/templates/left_sidebar.hbs

```diff
@@ -143,6 +143,15 @@
                     <span class="unread_count quiet-count"></span>
                 </a>
             </li>
+            <li class="top_left_change_navbar_banners top_left_row hidden-for-spectators">
+                <a class="left-sidebar-navigation-label-container">
+                    <span class="filter-icon">
+                        <i class="zulip-icon zulip-icon-spoiler" aria-hidden="true"></i>
+                    </span>
+                    {{~!-- squash whitespace --~}}
+                    <span class="left-sidebar-navigation-label">{{t 'Change navbar banner' }}</span>
+                </a>
+            </li>
```

Comment:
So this is cool and a thing I'd like to have available long term in some sort of "Zulip development tools" menu. We probably need to design such a menu in #design. But I think conceptually we want a menu that only appears if you've done something to trigger "developer mode" (say, a keyboard shortcut) and only works in the dev environment.

I assume we'll just not merge this commit when we merge the PR, so we're not blocked on figuring that out.

Reply:
The idea of having **developer mode** sounds exciting to me. I'd be up for handling this proposal and setting one up.